src/java/fr/paris/lutece/util/pool/PoolManager.java

```diff
@@ -136,7 +135,7 @@ private void createPools( Properties props ) throws LuteceInitException
         Enumeration propNames = props.propertyNames( );
         String strPoolName = "";
 
-        Hashtable<String, Hashtable<String, String>> htPools = new Hashtable<>( );
+        Map<String, Hashtable<String, String>> htPools = new HashMap<>( );

```

Comment:
Is it expected that the outer map is made un-synchronized, but not the inner map ?

Reply:
The outer map is purely a local variable so doesn't need to be synchronized. Actually I don't think that the inner map needs to be synchronized either but legacy code needs it to be a hashtable